### PR TITLE
Adds CIDR based subnet filters

### PR DESCRIPTION
**Pull request by hajati**
Upcoming release: `v3.3.0`

### CODE REVIEW

File: variables.tf [hunk 1]
```diff
@@ -510,6 +510,11 @@ variable "redis_subnet_tag_filter" {
   default     = {}
 }
 
+variable "redis_subnet_cidr_block_filter" {
```
Comment (snovikov) [inline variables.tf:513]: Please sepcify the type of variable explicitely

File: variables.tf [hunk 2]
```diff
@@ -812,6 +817,12 @@ variable "rds_subnet_tag_filter" {
   default     = {}
 }
 
+variable "rds_subnet_cidr_block_filter" {
```
Comment (snovikov) [inline variables.tf:820]: Please sepcify the type of variable explicitely

**Review by snovikov — CHANGES_REQUESTED** [review]
All variable have to have types defined.

File: variables.tf (hunk 1 above)
Comment (hajati) [reply variables.tf:513]: Done with [6232c30](https://github.com/Flaconi/terraform-aws-microservice/pull/62/commits/6232c305a6045017894e7f7d13a9f0bd647577d7).

File: variables.tf (hunk 2 above)
Comment (hajati) [reply variables.tf:821]: Done with [6232c30](https://github.com/Flaconi/terraform-aws-microservice/pull/62/commits/6232c305a6045017894e7f7d13a9f0bd647577d7).

**Review by snovikov — APPROVED** [review]

File: variables.tf (hunk 2 above)
Comment (danvaida) [reply variables.tf:821]: > Please sepcify the type of variable explicitely

Something that for us (DevOps team members) is seen as common sense, for others might not be entirely obvious. I'll open a PR here: https://github.com/Flaconi/devops-docs/blob/master/doc/conventions/style-guide-terraform.md

**Pull request merged**